single-line-diagram-core/src/main/java/com/powsybl/sld/layout/LayoutParameters.java

```diff
@@ -68,6 +68,8 @@
     private boolean labelCentered = false;
     private boolean labelDiagonal = false;
 
+    private boolean indicateOpenLines = true;
```

Comment:
After discussion with @jonenst it seems that highlightLineState would be more correct

Reply:
Parameter renamed